node_modules/oae-admin/tenants/tenants.html

```diff
@@ -121,8 +42,99 @@
+                <div class="col-xs-1">__MSG__STATUS__</div>
+            </li>
+        </ul>
+        <ul id="tenants-table-rows" class="oae-list"></ul>
+    </div>
+--></div>
+
+<div id="tenants-row-template"><!--
+    {var isOnGlobalAdminServer = context.isGlobalAdminServer || context.isTenantOnGlobalAdminServer}
+    {for tenant in results}
+        {var displayName = oae.api.util.security().encodeForHTMLAttribute(tenant.displayName)}
+        <li id="tenants-table-row-${tenant.alias|encodeForHTMLAttribute}" data-alias="${tenant.alias|encodeForHTMLAttribute}" class="row">
+            <div class="tenants-col tenants-col-edit">
+                {if context.isGlobalAdminServer}
+                    <a href="/tenant/${tenant.alias|encodeForURL}" title="__MSG__EDIT_TENANT__"><i class="fa fa-pencil"></i></a>
+                {/if}
+            </div>
+            <div class="col-xs-1">
+                ${tenant.alias|encodeForHTML}
+            </div>
+            {if oae.data.me.isGlobalAdmin}
+                <div class="jeditable-container col-xs-2">
+                    <div class="jeditable-field" data-alias="${tenant.alias|encodeForHTMLAttribute}" data-field="displayName">
+                        ${tenant.displayName|encodeForHTML}
+                    </div>
+                </div>
+            {else}
+                <div class="col-xs-1">
+                    ${tenant.displayName|encodeForHTML}
+                </div>
+            {/if}
+            <div class="tenants-host col-xs-4">
+                {if oae.data.me.isGlobalAdmin}
+                    <div class="tenants-visit">
+                        <a href="//${tenant.host}" title="__MSG__VISIT_TENANT__" target="_blank"><i class="fa fa-external-link tenants-visit"></i></a>
+                    </div>
+                    <div class="jeditable-container">
+                        <div class="jeditable-field" data-alias="${tenant.alias|encodeForHTMLAttribute}" data-field="host">
+                            ${tenant.host|encodeForHTML}
+                        </div>
+                    </div>
+                {else}
+                    <a href="//${tenant.host}" title="${tenant.displayName|encodeForHTMLAttribute}" target="_blank">
+                        ${tenant.host|encodeForHTML}
+                    </a>
+                {/if}
+            </div>
+            <div class="col-xs-1">
+                {if tenant.active}
+                    <span class="text-success">__MSG__RUNNING__</span>
+                {else}
+                    <span class="text-error">__MSG__DISABLED__</span>
+                {/if}
+            </div>
+            {if isOnGlobalAdminServer}
+                <div class="col-xs-1">
+                    {if tenant.active}
+                        <button type="button" class="btn btn-warning tenants-stop" data-alias="${tenant.alias|encodeForHTMLAttribute}" data-displayName="${tenant.displayName|encodeForHTMLAttribute}">__MSG__STOP__</button>
```

Comment:
Stopping the tenant causes a copy of the tenant to be appended to the list, rather than updating the existing item